static/app/components/metrics/mriSelect/index.tsx

```diff
@@ -315,7 +319,7 @@ export const MRISelect = memo(function MRISelect({
         isLoading={isLoading}
         loadingMessage={t('Loading\u2026')}
         menuSize="sm"
-        menuWidth="450px"
+        menuWidth="450px" // TODO(priscilawebdev): update this value for small screens
```

Comment:
since now we are using `useBreakpoints()` we can do better here and make this value more dynamic. Right now the menu does not look nice on small devices